Gems/Atom/Asset/ImageProcessingAtom/Code/Source/ImageLoader/QtImageLoader.cpp

```diff
@@ -57,6 +177,170 @@ namespace ImageProcessingAtom
             return pImage;
         }
 
+        // https://forum.qt.io/topic/74712/qimage-from-tga-with-alpha/11
+        // https://forum.qt.io/topic/101971/qimage-and-tga-support-in-c/5
+        // http://www.paulbourke.net/dataformats/tga/
+        bool LoadTgaImage(const AZStd::string& filename, QImage& image)
+        {
+            if (image.load(filename.c_str()))
+            {
+                return true;
+            }
+
+            // open the file
+            AZ::IO::SystemFile imageFile;
+            imageFile.Open(filename.c_str(), AZ::IO::SystemFile::SF_OPEN_READ_ONLY);
+
+            if (!imageFile.IsOpen())
+            {
+                AZ_Warning("Image Processing", false, "Failed to open file %s", filename.c_str());
+                return false;
+            }
+
+            AZ::IO::SystemFileStream imageFileStream(&imageFile, true);
+            if (!imageFileStream.IsOpen())
+            {
+                AZ_Warning("Image Processing", false, "Failed to create file stream %s", filename.c_str());
+                return false;
+            }
+
+            // read in the header
+            TgaHeader tgaHeader;
+            
+            if (imageFileStream.Read(sizeof(tgaHeader), &tgaHeader) != sizeof(tgaHeader))
+            {
+                AZ_Warning("Image Processing", false, "Failed to read file header %s", filename.c_str());
+                return false;
+            }
+
+            if (tgaHeader.m_dataTypeCode != ImageTypeCode::UnmappedRGB && tgaHeader.m_dataTypeCode != ImageTypeCode::RGBRLE)
+            {
+                AZ_Warning("Image Processing", false, "Unsupported type code [%d] of TGA file %s", tgaHeader.m_dataTypeCode, filename.c_str());
+                return false;
+            }
+
+            if (tgaHeader.m_bitsPerPixel != ImagePixelSize::Targa24 && tgaHeader.m_bitsPerPixel != ImagePixelSize::Targa32)
+            {
+                AZ_Warning(
+                    "Image Processing", false, "Unsupported bits per pixel [%d], type code [%d] of TGA file %s", tgaHeader.m_bitsPerPixel,
+                    tgaHeader.m_dataTypeCode, filename.c_str());
+                return false;
+            }
+
+            AZ::u32 bytesPerPixel = (AZ::u8)tgaHeader.m_bitsPerPixel / 8;
+            AZ::u32 imageBytesSize = tgaHeader.m_width * tgaHeader.m_height * bytesPerPixel;
+
+            AZStd::vector<AZ::u8> pixelContent;
+            pixelContent.resize(imageBytesSize);
```

Comment:
consider using resize_no_construct here.  We've had problems with it taking a long time to initialize u8 arrays for some reason.